Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```diff
@@ -1114,6 +1114,6 @@
     impactEffect: BulletImpactEffectRedDisabler
     damage:
       types:
-        Heat: 20 # Slightly more damage than the 17heat from the Captain's Hitscan lasgun
+        Heat: 25 # DeltaV buffed to 25 from 20
```

Comment:
this wont need a deltav comment after upstream merge, this is actually a deltav prototype only in this file because of cursed yml reference